plugins/dependencies/src/main/kotlin/dependencies/Koin.kt

```diff
@@ -0,0 +1,28 @@
+@file:Suppress("PackageDirectoryMismatch", "SpellCheckingInspection", "unused") // 1
+
+import de.fayard.refreshVersions.core.DependencyGroup
+import org.gradle.api.Incubating
+
+/**                                                             // 2
+ * A pragmatic lightweight dependency injection framework for Kotlin developers.
+ * Koin is a DSL, a light container and a pragmatic API
+ *
+ * - [Official website here](https://insert-koin.io/)
+ * - GitHub page: [Koin](https://github.com/InsertKoinIO/koin)
+ * - [GitHub Releases here](https://github.com/InsertKoinIO/koin/releases)
+ */
+@Incubating
+object Koin : DependencyGroup(
+    group = "io.insert-koin"
```

Comment:
Don't all Koin dependencies share the same version?
If yes, you should define here a `rawRule` similar to the ones here https://github.com/jmfayard/refreshVersions/tree/main/plugins/dependencies/src/main/resources/refreshVersions-rules

The file  `plugins/dependencies/src/test/resources/dependencies-versions-key-validated.txt` should then show `version.koin` instead of `version.NO-RULE`

Reply:
Will do, thanks!

Reply:
Would you mind changing the rule so that the key is `version.koin` insert of `version.insert-koin`?
Apart from that great job!